sdk/eventhub/azure-eventhubs/doc/azure.eventhub.exceptions.rst

```diff
@@ -0,0 +1,38 @@
+azure.eventhub.exceptions package
+=================================
+
+    .. autoclass:: azure.eventhub.exceptions.EventHubError
```

Comment:
This is hopefully a short term solution to avoid documenting things we want to remain hidden.
We should review whether we can go to full auto-doc for the next release.